test-packages/01-basic-app/app/models/shim.js

```diff
@@ -0,0 +1,10 @@
+// Mimics the static apis of ShimModelClass from ember-data
+export default class Shim {
+  fields;
+  attributes;
+  relationshipsByName;
+
+  eachAttribute() {}
+  eachRelationship() {}
+  eachTransformedAttribute() {}
+}
```

Comment:
When I add this file without the fix, this package's test suite fails
![image](https://github.com/miragejs/ember-cli-mirage/assets/36919/ddba462a-df03-43e6-b311-e57d59c49533)